builder.go

```diff
@@ -202,6 +211,7 @@ func isBuild(path string) bool {
 }
 
 type processMap map[string]Process
+type sliceList []Slice
```

Comment:
I don't think we need this type. `[]Slice` is explicit already at it is symmetrical with `[]BOMEntry`.